packages/core-ui/lib/core/index.tsx

```diff
@@ -226,3 +227,29 @@ export const ButtonRow = ({ children, style, yMargin }: RowProps) => (
     {children}
   </div>
 );
+
+/* RowPrimaryOrRowSecondary
+ *
+ * The primary/secondary module in a PageRow.
+ */
+export const RowPrimaryOrRowSecondary = ({
```

Comment:
Why have you merged `RowPrimary` and `RowSecondary` into one component?

Please separate as per what is in the staking dashboard. We can then look at optimisations after the code has moved to this repo.